server.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/juju/bundlechanges"
+	"github.com/juju/bundleservice/params"
+	"gopkg.in/juju/charm.v6-unstable"
+
+	"github.com/juju/httprequest"
+	"github.com/julienschmidt/httprouter"
+)
+
+// main builds and runs the server when the run command is called.
+func main() {
+	router := httprouter.New()
+	// Add handlers
+	f := func(p httprequest.Params) (*handler, error) {
+		return &handler{}, nil
+	}
+	for _, h := range errorMapper.Handlers(f) {
+		router.Handle(h.Method, h.Path, h.Handle)
+	}
+	log.Fatal(http.ListenAndServe(":8000", router))
+}
+
+// The handler type contains all of the handlers for the server.
+type handler struct{}
+
+// errorResponse represents an error encountered by the server.
+type errorResponse struct {
+	Message string
+}
+
+// errorMapper maps an error from a handler into an HTTP server error.
+var errorMapper httprequest.ErrorMapper = func(err error) (int, interface{}) {
```

Comment:
+1 on the TODO.

You could use httprequest.RemoteError instead of errorResponse here;
that way you know you're using the standard error form.